cypress/integration/common/report.js

```diff
@@ -169,7 +169,7 @@ Then('each list item links to the post page', () => {
 Then('I can visit the post page', () => {
   cy.contains(annoyingUserWhoMutedModeratorTitle).click()
   cy.location('pathname').should('contain', '/post')
-    .get('h3').should('contain', annoyingUserWhoMutedModeratorTitle)
+    .get('.ds-card-content .hyphenate-text').should('contain', annoyingUserWhoMutedModeratorTitle)
```

Comment:
Code creep?

Reply:
Could be, yes.
It failed locally, but that was in headless. Also, I was reading cypress docs and they suggest to never use tags as selectors.
We could open another PR to clean up our cypress code though